setup.py

```diff
@@ -22,15 +22,15 @@ def package_data(pkg, roots):
 
 setup(
     name='xblock-eoc-journal',
-    version='0.9.4',
+    version='0.9.5',
```

Comment:
Can you bump this to '0.10.0' instead as we are dropping Python2 support?

Reply:
Addressd!